src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/MethodConfigurationComponent.scala

```diff
@@ -152,13 +160,25 @@ trait MethodConfigurationComponent {
                 where mc.workspace_id=$workspaceId"""
         })
       }
+
+      def hideAction(methodId: Long) = {
+        val now = DateTime.now.toString("yyyy-MM-dd_HH:mm:ss")
+        sqlu"""UPDATE METHOD_CONFIG mc
```

Comment:
why do we need raw sql for this? This can be done easily with slick and then we don't need a DeleteMethodConfigurationQuery object